public/static/locales/en/common.json

```diff
@@ -90,6 +90,7 @@
     "update": "Update",
     "updateAccountInformation": "Update Account Information",
     "updateNow": "Update Now",
+    "usageSummaryError": "Unable to get usage summary.",
```

Comment:
Moving this message to the `common` ns prevents other pages from requiring the entire `dashboard` ns for only this message.